app/views/comments/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Showing all comments:</h1>
```

Comment:
Is this a scaffold generated file? I haven't used scaffolds yet but I'm guessing that's what's happening here and then when we learned one to many you changed it? If so maybe delete some of these views/comments files. If not, my bad!